wiki/Rules/Content_Usage_Guidelines/en.md

```diff
@@ -0,0 +1,39 @@
+
+To ensure a user's account remains in good standing, it is recommended to create beatmaps using assets that are any of the following:
+
+- legally cleared for use in osu!
+- permitted for use by their creator(s)
+- copyright-free
+
+## Audio
+
+osu! provides its userbase a library of licensed songs via the [Featured Artist](/wiki/Featured_Artists) program. Each song on this listing is free for use within osu! and has no risk of removal.
+
+When looking for songs outside of osu!'s featured artist catalog, check whether or not its creator has explicitly stated usage terms. This information can often be found in description sections of an artist's music release platforms (e.g. Bandcamp, Soundcloud, YouTube) or their personal website. If a song is licensed under a [Creative Commons License](https://creativecommons.org/licenses/by-nc-sa/3.0/) or is available for non-commercial use, it is safe to create and submit a beatmap for it.
+
+If you want to create a beatmap for a song and its usage rights are not clear, it is recommended to reach out to its artist for permission. Most artists have contact methods labeled on their music release platforms, personal websites, and social media.
+
+While music is frequently copyright-restricted, sound effects used for custom hitsounds are more often freely available. For example, the Philharmonia orchestra provides free [instrument sound samples](https://philharmonia.co.uk/resources/sound-samples/) and [Freesound](https://freesound.org) permits sound sample searches filtered by usage rights.
+
+## Visual
+
+Visual assets include a beatmap's background/storyboard/video and follow the same usage requirements as audio assets.
+
+As with music, it is recommended to seek out an artist's input on usage terms and to ask for permission when those terms are unclear. For example, if you're interested in using a background from an artist on pixiv, locate their profile for usage terms or send them a message asking for permission.
+
+Many image-hosting websites have search filters for Creative Commons Licensing or non-commercial use. Some websites that mappers have frequently used to find their copyright-free backgrounds include:
+
+- [pixabay](https://pixabay.com/)
+- [Unsplash](https://unsplash.com/)
+- [Pexels](https://www.pexels.com/)
+
+If you've received a musician's permission to use their song, their album artwork or any other official artwork for the artist is also likely viable for use as a background/storyboard. Again however, ask for permission if this is not clear.
+
+## Gameplay
+
+When submitting a beatmap, you should have permission for gameplay-related assets such as [guest difficulties](/wiki/Beatmaps/Beatmapsets/Guest_difficulty) or charts from other rhythm games.
```

Comment:
```suggestion
When submitting a beatmap, you should have permission for gameplay-related assets such as [guest difficulties](/wiki/Beatmaps/Beatmapsets/Guest_difficulty) or charts from other rhythm games.

We do not condone submissions featuring gameplay (hit objects) matching that of other games, as this is not respectful to the rhythm game community.
```

I think it might be good to explicitly state this